content/docs/ops/secrets.md

```diff
@@ -157,3 +161,41 @@ All UAA clients and users and associated credentials should be created via the C
     ```sh
     rm -rf tmp
     ```
+
+#### Updating Concourse Secrets
+
+Updating Concourse secrets combines the shared secrets above with local config to change a Concourse pipeline.
+
+1. Clone the [cg-scripts](https://github.com/18F/cg-scripts) repository.
+
+    ```sh
+    git clone https://github.com/18F/cg-scripts.git
+    ```
+
+1. If you don't already have the AWS CLI set up with credentials, see [the steps above](#aws-credentials).
+
+1. `cd` to the local clone of the Concourse pipeline you are updating
+
+1. Create a `concourse-environment` file.
+
+    ```sh
+    aws-vault exec cloud-gov-govcloud bash
+    CI_ENV=cloud-gov-govcloud SECRETS_BUCKET=cloud-gov-varz CG_PIPELINE=~/cg-pipeline-tasks ~/cg-scripts/generate-concourse-environment.sh cg-deploy-bosh.yml
+    ```
+
+    Where `CI_ENV` is the environment configured for local Concourse `fly` CLI, `SECRETS_BUCKET` is the encrypted S3 bucket where credentials are stored, and `CG_PIPELINE` is the path to a local clone of the `cg-pipeline-tasks` git repository.
+
+    Any number of local yml files (`cg-deploy-bosh.yml`) can also be merged to create the final `concourse-environment.yml`.
+
+1. Update the Concourse pipeline with `fly`.
+
+    ```sh
+    fly -t cloud-gov-govcloud set-pipeline -p deploy-bosh -c pipeline.yml -l concourse-environment.yml
+    ```
+
+1. Don't leave the secrets lying around (for security reasons, and because they get stale).
+
+    ```sh
```

Comment:
Would it be overkill to recommend `mktemp` or `mktemp -d`? Or maybe begin each example with `cd $(mktemp -d)`? I would never do this, but it's possible that we might forget to clean up files after doing secrets work.

Reply:
👍